pyseir/ensembles/ensemble_runner.py

```diff
@@ -94,27 +194,27 @@ def run_ensemble(self):
         Run an ensemble of models for each suppression policy nad generate the
         output report / results dataset.
         """
+        for suppression_policy_name, suppression_policy in self.suppression_policies.items():
+            logging.info(f'Running simulation ensemble for {self.county_metadata["county"]}, {self.county_metadata["state"]} {suppression_policy_name}')
```

Comment:
Can you use a module level logger? 
```
import logging
_logger = logging.getLogger(__name__)
```

Reply:
Yes, but if you don't mind let me do this tomorrow across all modules.  Do you have a specific preference for why? We currently output filename and line no in the root logger.

Reply:
Mostly because it gives the user of the package more granular log control.  I'll sometimes set logging levels at different levels depending on the package:
for example:
```
logging.getLogger("botocore").setLevel(logging.INFO)
logging.getLogger("boto3.resources").setLevel(logging.INFO)
logging.getLogger("urllib3.connectionpool").setLevel(logging.INFO)
```
if its on the root logger you don't get that control